### PR TITLE
Display subrooms in the join room tab

### PR DESCRIPTION
**Pull request by bumbadadabum**
Requires https://github.com/Zarel/Pokemon-Showdown/pull/4014

### CODE REVIEW

File: js/client-rooms.js [hunk 1]
```diff
@@ -60,7 +60,7 @@
 		},
 		renderRoomBtn: function (roomData) {
 			var id = toId(roomData.title);
-			return '<div><a href="' + app.root + id + '" class="ilink"><small style="float:right">(' + Number(roomData.userCount) + ' users)</small><strong><i class="fa fa-comment-o"></i> ' + Tools.escapeHTML(roomData.title) + '<br /></strong><small>' + Tools.escapeHTML(roomData.desc || '') + '</small></a></div>';
+			return '<div><a href="' + app.root + id + '" class="ilink"><small style="float:right">(' + Number(roomData.userCount) + ' users)</small><strong><i class="fa fa-comment-o"></i> ' + Tools.escapeHTML(roomData.title) + '<br /></strong><small>' + Tools.escapeHTML(roomData.desc || '') + (roomData.subRooms.length ? '<br/><strong>Subrooms: ' + roomData.subRooms.join(', ') + '</strong>' : '') + '</small></a></div>';
```
Comment (panpawn) [inline js/client-rooms.js]: This should probably be escaped - what if a subroom's title contains HTML?

**Comment by Zarel** [comment]
Weird, the retest didn't rebase? You might need to manually rebase.

**Comment by Zarel** [comment]
You still need to fix HTML escaping, anyway.

**Comment by bumbadadabum** [comment]
Updated.

**Pull request merged**